test/micro/org/openjdk/bench/java/util/RandomGeneratorNextBytes.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package org.openjdk.bench.java.util;
+
+import java.util.concurrent.TimeUnit;
+import java.util.random.RandomGenerator;
+
+import org.openjdk.jmh.annotations.*;
+
+@BenchmarkMode(Mode.Throughput)
+@OutputTimeUnit(TimeUnit.MILLISECONDS)
+@State(Scope.Thread)
+@Warmup(iterations = 5, time = 3)
+@Measurement(iterations = 4, time = 2)
+@Fork(value = 3)
+public class RandomGeneratorNextBytes {
+
+    @Param({"Random", "L32X64MixRandom"})
+    private String algo;
+
+    @Param({"1", "2", "3", "4", "6", "7", "9", "10", "48", "512", "1000"})
```

Comment:
Why is there no `8` `16`? I don't think we need this many cases of small sizes as byte array arguments should usually have large sizes.

Reply:
> Why is there no `8` `16`? I don't think we need this many cases of small sizes as byte array arguments should usually have large sizes.

Sorry, I didn't think about which length to use at the time, so I just copied other benchmarks and went to bed.